### PR TITLE
Create another Way To Print Hello World

### PR DESCRIPTION
**Pull request by Madhur-Jatiya**

### CODE REVIEW

File: java/anotherWayToPrintHelloWorld.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+public class WaysToPrintHello {
```
Comment (Dimas-Saputra-Me) [inline java/anotherWayToPrintHelloWorld.java:3]: Please add file information as stated in template, like this (on the top of the file)
```sh
// Title  : [Title] 
// Author : [Your-Profile-Username](Your-Profile-Link)
```

**Review by Dimas-Saputra-Me — CHANGES_REQUESTED** [review]
Please add one more thing stated below, on your code file. Thank you

**Comment by Madhur-Jatiya** [comment]
now i add all things you required plz merge my pull request.

**Comment by Dimas-Saputra-Me** [comment]
**\~Thank You\~**

**Review by Dimas-Saputra-Me — APPROVED** [review]

**Pull request merged**